backend/src/schema/types/type/Post.gql

```diff
@@ -171,12 +175,30 @@ type Post {
     @cypher(statement: "MATCH (this)<-[emoted:EMOTED]-(:User) RETURN COUNT(DISTINCT emoted)")
 
   group: Group @relation(name: "IN", direction: "OUT")
+
+  postType: [PostType]
+    @cypher(statement: "RETURN filter(l IN labels(this) WHERE NOT l = 'Post')")
```

Comment:
Please explain this to me - you keep filtering this out - why?

Reply:
All user contributions have the label `post`. Then they get a second label, `Event` or `Article`. `postType` therefore is either `Article` or `Event`. So I get all labels and filter out the post label